toolkit/docs/building/building.md

```diff
@@ -569,7 +568,7 @@ These are the useful build targets:
 | clean-*                          | Most targets have a `clean-<target>` target which selectively cleans the target's output.
 | compress-rpms                    | Compresses all RPMs in `../out/RPMS` into `../out/rpms.tar.gz`. See `hydrate-rpms` target.
 | compress-srpms                   | Compresses all SRPMs in `../out/SRPMS` into `../out/srpms.tar.gz`.
-| copy-toolchain-rpms              | Copy all toolchain RPMS from `../build/rpm_cache/cache` to  `../out/RPMS`.
+| copy-toolchain-rpms              | **Generally no longer needed** Copy all toolchain RPMS from `../build/rpm_cache/cache` to  `../out/RPMS`.
```

Comment:
Could we remove its usage from our pipelines in that case? This would be a good test if it's really no longer needed.

Reply:
Not easily, since we still want to support old versions of 2.0 and all 1.0 builds. I'd also like to not break downstream builds that may use it. Maybe for 3.0 we can fully drop it since that is a good if/else condition that we probably already are going to be using.

Reply:
You're right, that it's not trivial. But in that case I'd maybe use this wording to send a clear message of our intentions:
```suggestion
| copy-toolchain-rpms              | **DEPRECATED** Copy all toolchain RPMS from `../build/rpm_cache/cache` to  `../out/RPMS`.
```

Reply:
Added a few details: `**[DEPRECATED]: This should no longer be needed as a work around in core repo builds. Will be removed in future versions.** Copy all toolchain RPMS from `../build/toolchain_rpms` to  `../out/RPMS`.`